src/components/organisms/Badges/BadgeImage.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {
+  RefObject,
+  useCallback,
+} from "react";
+
+export const BadgeImage: React.FC<{
+  badgeImgRefs: RefObject<HTMLImageElement[]>;
```

Comment:
It looks like we're mutating an array passed in here? That sort of breaks React's 'single direction flow' with props/etc. Is there a better way we can achieve this?

Reply:
badgeImgRefs has been removed because each component can now keep it's own ref and this is unnecessary.

Reply:
Sounds good!